packages/common/src/hooks/useRowRenderCount/useRowRenderCount.ts

```diff
@@ -11,9 +11,14 @@ export const useRowRenderCount = (): number => {
   const dataRowHeight = mixins.table.dataRow.height;
   const headerRowHeight = mixins.table.headerRow.height;
   const paginationRowHeight = mixins.table.paginationRow.height;
-
+  const pageFooterHeight = mixins.footer.height;
```

Comment:
it does reduce the rows to display unfortunately. 
the problem is just that the table squashes the footer so that the items don't vertically align